.github/workflows/node-test.yml

```diff
@@ -0,0 +1,27 @@
+name: Node.js CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{matrix.os}}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node-version: [12.x]
```

Comment:
Am I right that you can't use `LTS` and have to pick a version here?

We'll have to remember to update this down the road...

Reply:
Sorry, I wasn't clear.  I meant for node-version.

Reply:
Thanks for confirming with me--I'm new to GitHub Actions.

R+ from me on this, let's try this.